explain/setup/google_cloud_sql/01_auto_explain_check.mdx

```diff
@@ -0,0 +1,21 @@
+---
+title: 'Step 1: Check for auto_explain'
+install_track_title: 'Automated EXPLAIN: Setup (Google Cloud SQL)'
+backlink_href: /docs/explain/setup
+backlink_title: 'Automated EXPLAIN: Setup'
+---
+
+import Step01InApp from './_01_in_app.mdx'
+import Step01Public from './_01_public.mdx'
+
+The recommended Automated EXPLAIN configuration for Google Cloud SQL is to use
+the Postgres `auto_explain` contrib module. This ensures accurate and detailed plan
+information, including runtime information (as gathered by EXPLAIN ANALYZE). With our
+recommended configuration (presented in a later step), performance overhead is minimal.
+
```

Comment:
It looks like the step components included have RDS-specific wording, probably good to fix:

"First, to check if auto_explain is already enabled on your system, run the following query as an **RDS superuser**:"

Reply:
This is actually an instance of the stupid Gatsby MDX behavior I was complaining about. The components are right, but because they share the same name (in totally different files!) there's a collision and the wrong one is pulled in. Good catch.

Reply:
Oh. Sigh. *yells at cloud*